configs/components/marionette-collective.rb

```diff
@@ -66,7 +66,9 @@
   when "aix"
     pkg.install_service "resources/aix/mcollective.service", nil, "mcollective"
   when "windows"
-    puts "Service files not enabled on windows"
+    # Sample - service definition for rubyw.
+    # 2nd argumenent is the Wix service specific file to be included.
+    pkg.install_service "stagedir\\ruby\\bin\\rubyw.exe", "resources/windows/msi/service.mcollective.wxs"
```

Comment:
Its more a sample of how we will use it and gives me a direction when coding the actual platform specific install_service.